keps/sig-architecture/2019-03-19-go-modules.md

```diff
@@ -167,13 +160,29 @@ See the [alternatives](#alternatives-to-publishing-staging-component-modules) se
 
 As part of transitioning to go modules, we can select a versioning strategy.
 
-Current state:
-* `client-go` tags a major version on every kubernetes release
-* other components tag a (non-semver) `kubernetes-1.x.y` version on each release
-* no import rewriting occurs
-* consumers can only make use of a single version of each component
+State prior to adoption of go modules:
+* `client-go` tagged a major version on every kubernetes release
+* all components tagged a (non-semver) `kubernetes-1.x.y` version on each release
+* no import rewriting occured
+* consumers could only make use of a single version of each component in a build
+
+State after adoption of go modules:
+* all components tagged a (non-semver) `kubernetes-1.x.y` version on each release
+* no import rewriting occured
+* consumers could only make use of a single version of each component in a build
```

Comment:
Calling out these limitations followed by a proposal raises the expectation that the proposal somehow addresses them (at least it did for me). But it only addresses the first point, right? Even with `v0.x.y` tags consumers will still be limited to one version of each component (@msau42, @xing-yang).

Not sure whether that's worth clarifying in the document.

Reply:
This is explicitly called out below:
> This results in the following usage patterns:
> ...
> * consumers are limited to a single copy of kubernetes libraries among all dependencies (as they are today)